package.json

```diff
@@ -7,64 +7,65 @@
   "private": true,
   "dependencies": {
     "@types/enzyme": "3.10.3",
-    "@types/history": "^4.7.3",
-    "@types/jest": "24.0.17",
+    "@types/history": "4.7.3",
+    "@types/jest": "24.0.21",
     "@types/lodash": "4.14.144",
     "@types/mark.js": "8.11.4",
     "@types/moment-timezone": "0.5.12",
-    "@types/node": "12.7.0",
+    "@types/node": "12.12.5",
     "@types/react": "16.8.24",
     "@types/react-bootstrap": "0.32.20",
     "@types/react-dom": "16.8.5",
-    "@types/react-router-dom": "4.3.4",
-    "@types/react-transition-group": "4.2.2",
-    "@typescript-eslint/parser": "1.13.0",
+    "@types/react-router-dom": "5.1.2",
+    "@types/react-transition-group": "4.2.3",
+    "@typescript-eslint/parser": "2.6.0",
```

Comment:
Should these types be listed as `devDependencies` instead so that during production installation, those will not be installed?

Reply:
@saturninoabril - good idea. done